mungegithub/mungers/cla.go

```diff
@@ -113,14 +125,27 @@ func (c *ClaMunger) Munge(obj *githubhelper.MungeObject) {
 		return
 	}
 
-	if obj.HasLabel(cncfClaNoLabel) {
-		// status reported error/failure and we've already applied 'cncf-cla: no' label.
+	// If we are here, that means that the context is failure/error.
+	comments, err := obj.ListComments()
+	if err != nil {
+		glog.Error(err)
 		return
 	}
+	startDate := c.LastComment(comments, c.MungerNotificationName(claNagNotifyName), nil)
```

Comment:
I think that doesn't do what you want (or I mean, this should is done automatically)